paket.dependencies

```diff
@@ -13,9 +14,8 @@ github fsprojects/FSharp.TypeProviders.SDK tests/ProvidedTypesTesting.fs
 
 group Test
     source https://api.nuget.org/v3/index.json
-    framework: net6.0
 
     nuget FSharp.Core redirects: force
-    nuget Expecto
-    nuget System.Configuration.ConfigurationManager redirects: force
+    nuget Expecto 9.0.4
```

Comment:
just wonder why do you need to pin Expecto version?

Reply:
Expecto 10 didn't have [runTestsInAssembly](https://github.com/fsprojects/FSharp.Configuration/blob/084f63d8a70f4e9f56e3c09aa5362920884781a0/tests/FSharp.Configuration.Tests/Program.fs#L5C5-L5C23).

Reply:
I think you can change it to 

```runTestsInAssemblyWithCLIArgs [] args``` 

to use Expecto 10+, though newer versions don't support .NET Framework any more so you might need to stick with the old version to multi-target the tests to that.